travis/build_and_run_all_spark-3.0_hadoop-3.2.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
```

Comment:
The name includes `build`, please put mvn build command here

Reply:
@gczsjdy  Thanks , I will move here.